docs/02.API-REFERENCE.md

```diff
@@ -450,9 +467,9 @@ jerry_register_magic_strings  (const jerry_char_ptr_t *ex_str_items_p,
 
   // must be static, because 'jerry_register_magic_strings' does not copy
   static const jerry_length_t magic_string_lengths[] = {
-                                                         (jerry_length_t)strlen (magic_string_items[0]),
-                                                         (jerry_length_t)strlen (magic_string_items[1]),
-                                                         (jerry_length_t)strlen (magic_string_items[2])
+                                                         12,
```

Comment:
`magic_string_lengths` is a `static const` array variable, it cannot be initialized with the return value of a function call, C99 does not allow that. The extracted code did not compile.